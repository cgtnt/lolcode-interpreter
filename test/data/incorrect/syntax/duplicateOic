HAI 

BOTH SAEM 1 AN 2, O RLY?
  YA RLY
      NO WAI
    VISIBLE "OK"
OIC
OIC

KTHXBYE
