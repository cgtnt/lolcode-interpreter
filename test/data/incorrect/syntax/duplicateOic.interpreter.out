Line 8: Invalid expression, unexpected token OIC
